src/microbe_stage/MicrobeHUD.cs

```diff
@@ -623,7 +635,10 @@ private void UpdateHoverInfo()
                 var compoundIcon = GUICommon.Instance.CreateCompoundIcon(readableName, 25, 25);
 
                 var compoundsText = new StringBuilder(readableName, 150);
-                compoundsText.AppendFormat(CultureInfo.CurrentCulture, ": {0:F1}", entry.Value);
+
+                // : {0:F1}
+                compoundsText.AppendFormat(CultureInfo.CurrentCulture,
+                    TranslationServer.Translate("MICROBE_HUD_COUMPOUNDS_VAL"), entry.Value);
```

Comment:
I don't think translating just a number conversion is needed.
What should be translated is the `readableName` a couple lines above this.